From b630245291fd0534e1a7d31b643e1ebce7ef07a0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jo=C3=A3o=20Henrique=20Ferreira=20de=20Freitas?=
 <joaohf@gmail.com>
Date: Fri, 7 Jun 2024 21:39:12 -0300
Subject: [PATCH] Fix implicit-int

With GCC 14 series, the following error exists:

dead_port.c:43:26: error: return type defaults to 'int' [-Wimplicit-int]
   43 | #define MAIN(argc, argv) main(argc, argv)
      |                          ^~~~
dead_port.c:50:1: note: in expansion of macro 'MAIN'
   50 | MAIN(argc, argv)
      | ^~~~
---
 erts/emulator/test/port_SUITE_data/dead_port.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/erts/emulator/test/port_SUITE_data/dead_port.c b/erts/emulator/test/port_SUITE_data/dead_port.c
index 26f09f33c7..723f0bc67d 100644
--- a/erts/emulator/test/port_SUITE_data/dead_port.c
+++ b/erts/emulator/test/port_SUITE_data/dead_port.c
@@ -40,7 +40,7 @@
 #include "winbase.h"
 #endif
 
-#define MAIN(argc, argv) main(argc, argv)
+#define MAIN(argc, argv) int main(argc, argv)
 
 extern int errno;
 
